README.md

```diff
@@ -1,24 +1,37 @@
+# Cheap Crappy Constant Current Portable Programmable Power Supply 
 
-# About
-> We have `programable power supply` at home.
+`CCCCPPPS` is a DIY Programmable Buck/Boost Power Supply capable of  
+* Max Output Voltage  16V
+* TBD
 
 # Usage
 ## Debug Interface:
-This interface is enabled by default when building the firmware
+This interface is enabled by default when building the firmware. Use `minichlink`
+to issue commands listed below
+
 ## USB Interface:
 Build with `-DCONFIG_USE_USB`
 > [!NOTE]
 > To be implemented.
 
 ## Commands
- - `0` to turn the supply off
- - `c` and `v` to switch between constant current and constant voltage
- - `+` or `=` to increse the target voltage/current in 50mV/25mA increments
- - `1-9` to quickly set the voltage/current target in multiples of 1000mV/100mA
- - ~~Konami code makes the unit self distruct.~~ Feature removed due to misuse.
+> [!NOTE]
+> Need to improve the command list and handler
+
+- `0` to turn the supply off
+- `1-9` to set the target Voltage/Current in multiples of 1000mV/100mA
+- `c` to switch to Constant Current Mode 
+- `v` to switch to Constant Voltage Mode
+- `+` to increase the target Voltage/Current depending on mode, in 50mV/25mA increments
```

Comment:
forgot `-` to decrease.
Should we even mention this? its only for debugging without usb.